src/library.js

```diff
@@ -1409,21 +1409,21 @@ LibraryManager.library = {
 
   _emscripten_throw_longjmp__sig: 'v',
   _emscripten_throw_longjmp: function() { throw 'longjmp'; },
-#if !SUPPORT_LONGJMP
+#if SUPPORT_LONGJMP == 'none'
 #if !INCLUDE_FULL_LIBRARY
   // These are in order to print helpful error messages when either longjmp of
   // setjmp is used.
   longjmp__deps: [function() {
-    error('longjmp support was disabled (SUPPORT_LONGJMP=0), but it is required by the code (either set SUPPORT_LONGJMP=1, or remove uses of it in the project)');
+    error('longjmp support was disabled (SUPPORT_LONGJMP=none), but it is required by the code (either set SUPPORT_LONGJMP=emscripten, or remove uses of it in the project)');
```

Comment:
Do we really want to advertise these new settings.. maybe we can keep advertising the old =0 and =1 settings?  Ideally one day we will change the meaning of `=0` and `=1` so maybe they are still useful.. and maybe we shouldn't consider them legacy, but more like default on/off?

Reply:
Then what do you want each of the values to represent eventually? `0` means no support and `1` means Wasm SjLj, and we don't provide Emscripten option?

Reply:
Done